src/validations.ts

```diff
@@ -1,4 +1,5 @@
-const emailRegex = /^([a-zñ\d[\]])(\.?([\wñ&/~\-+\][]+))*@([ñ\w.\-\][]+)\.[\w\-\][.]{2,}(\?)?.*$/i;
+// eslint-disable-next-line
```

Comment:
Why?

Reply:
We saw with @cbernat there is infinite loop with the previews one and found one more new and works with more escenarios and cases

Reply:
I was talking about the disabled rule. What is the rule? Why does it cannot be fixed?

Reply:
in the library has too maybe becuse this warning ```
Unexpected control character(s) 
in regular expression: \x09, \x0d, \x0a, \x09,   
\x01, \x08, \x0b, \x0c, \x0e, \x1f, \x01, \x09,   \x0b, \x0c, \x0d, \x09, \x0d, \x0a, \x09
```

Reply:
mmm, and what do you think about those characters? is eslint wrong? do we really need them?

Reply:
if we want to support more cases yes,